pkg/api/index.go

```diff
@@ -76,7 +68,7 @@ func (hs *HTTPServer) setIndexViewData(c *contextmodel.ReqContext) (*dtos.IndexV
 		settings.AppSubUrl = ""
 	}
 
-	navTree, err := hs.navTreeService.GetNavTree(c, hasEditPerm, prefs)
+	navTree, err := hs.navTreeService.GetNavTree(c, prefs)
```

Comment:
note: the AC check that was passed on here, was only used at [buildDashboardNavLinks](https://github.com/grafana/grafana/pull/68995/files#diff-ac635e7d737f4ee23844f3fea08bd5458afd46d01f6378f17b3ff26a69d9dcdaR370), and was redundant, as we also check for the perm to create dashboards there.